### DISCUSSION TITLE
Regarding pop push

### DISCUSSION THREAD

**Discussion by AdityaisHacker** [community/community#132250] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

how to we do push and ppop in stack and queue data structure

**Comment by Anamicca23** [comment, marked as answer]
by using stack.pop(); for pull elemnt outsite and stack.push_back(); for pushing element back in stack.

**Comment by ebndev** [comment]
Hi @AdityaisHacker & @Anamicca23, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
